src/ledger/backend/tokencache.ts

```diff
@@ -20,10 +20,31 @@ interface Responder {
 	symbol(): Promise<string>;
 }
 
+export interface TokenCache {
+	readonly typeCache: TokenTypes;
+	readonly holderCache: TokenHolders;
```

Comment:
Why do we expose them in the interface? Isn't this an implementation detail?

Reply:
I left them in initially because I was refactoring. The next step should have been to get rid of them and fix any upcoming errors, but I seemingly forgot about that and instead decided a leaky abstraction is better :laughing: 

Fixed :rocket: